pkg/cache/server/config.go

```diff
@@ -87,7 +88,7 @@ func (c *Config) Complete() (CompletedConfig, error) {
 	}}, nil
 }
 
-func NewConfig(opts *cacheserveroptions.CompletedOptions) (*Config, error) {
+func NewConfig(opts *cacheserveroptions.CompletedOptions, restConfig *rest.Config) (*Config, error) {
```

Comment:
can we qualify that more? Which restConfig is that? To the shard?

Reply:
yes, `optionalShardConfig` when we run in-process, will update the name.